packages/rum/test/userActionCollection.spec.ts

```diff
@@ -92,6 +99,146 @@ describe('startUserActionCollection', () => {
     button.click()
 
     clock.expire()
+  }
+
+  it('cancels ongoing user action on view loading', () => {
+    const fakeLocation: Partial<Location> = { pathname: '/foo' }
+    const mockView: Partial<View> = {
+      documentVersion: 0,
+      id: 'foo',
+      location: fakeLocation as Location,
+    }
+    button.addEventListener(DOM_EVENT.CLICK, () => {
+      clock.tick(BEFORE_USER_ACTION_VALIDATION_DELAY)
+      // Since we don't collect dom mutations for this test, manually dispatch one
+      lifeCycle.notify(LifeCycleEventType.DOM_MUTATED)
+    })
+
+    clock.tick(SOME_ARBITRARY_DELAY)
+    button.click()
+    lifeCycle.notify(LifeCycleEventType.VIEW_COLLECTED, mockView as View)
+    clock.expire()
+
+    expect(events).toEqual([])
+  })
+
+  it('starts a user action when clicking on an element during a view loading', () => {
```

Comment:
I don't think this test is relevent anymore, since we don't do anything special related to user actions after a new view is created.